package.json

```diff
@@ -40,10 +40,11 @@
     "@nextcloud/auth": "^1.3.0",
     "@nextcloud/axios": "^1.6.0",
     "@nextcloud/dialogs": "^3.1.2",
+    "@nextcloud/event-bus": "^2.0.0",
     "@nextcloud/paths": "^2.0.0",
     "@nextcloud/router": "^2.0.0",
     "@nextcloud/vue": "^3.9.0",
-    "@skjnldsv/vue-plyr": "^7.0.1-alpha.0",
+    "@skjnldsv/vue-plyr": "7.0.1-alpha.0",
```

Comment:
?

Reply:
Yeah, sorry forgot to explain. It auto-updates to the beta, which breaks the build because of a missing css file in the build.

Reply:
Are you building with `npm i` or `npm ci` ?

Reply:
Both, `npm i` to install the new dep, which bumps this package in `json-lock` + `npm ci` before the build.

I can try again if you prefer to keep this dep unchanged. But I already tried a few time

Reply:
npm ci will install the exact listed version in the lock file. You don't need to edit the package.json

Reply:
ps: we never use npmi unless we're managing/maintaining dependencies versions